CharacterCreation/04_00_WeaponAttachments.txt

```diff
@@ -431,6 +431,18 @@ Takes up twice the space in your magazine inventory as a standard magazine.
 Only applicable to box-fed automatic weapons [not belt-fed], SMGs, 
 Carbines, and shotguns.
 
+== Baffled Ammo ($1.5 x mag cost) ==
+
+    This sabot-discarding ammo has baffled edges so that it barely makes any
+noise as it travels through the air. There is no zip or snap noise as bullets
+whiz by or into targets. 
+
+    Using Baffled ammo in your weapon reduces the range of your R3 bracket by
+10m and reduces the damage of your weapon by 5 damage. Stealth rolls to hide
+silent weapons fire automatically succeed. A bullet impact can be heard if a
+character is within 5m of the impact, actively curious and makes a search check

```

Comment:
Search check -> Awareness save?

I would also consider 'alert' over 'actively curious'. Making it less impossible for the enemy to notice the shots at close range adds to the commando puzzle by incentivizing taking out isolated targets first.

Reply:
Agreed on both points. Update pending.